CJBCheatsMenu/i18n/default.json

```diff
@@ -56,6 +56,8 @@
     "farm.instant-build": "Instant Build",
     "farm.always-auto-feed": "Always Auto-Feed",
     "farm.infinite-hay": "Infinite Hay",
+    "farm.no-decay": "No Animal Friendship Decay",
```

Comment:
This doesn't fully prevent friendship decay, since not petting them is [only one cause of decay](https://stardewvalleywiki.com/Animals#Friendship). Since fully preventing decay is probably out of scope for this PR, I suggest renaming the feature:
```json
"farm.always-auto-pet": "Always Auto-Pet Animals"
```